services/server/src/sourcify-chains-default.json

```diff
@@ -1381,6 +1381,17 @@
       }
     }
   },
+  "1030": {
+    "sourcifyName": "Conflux eSpace",
+    "supported": true,
+    "etherscanApi": {
+      "apiURL": "https://evmapi.confluxscan.net",
+      "apiKeyEnvName": "CONFLUXSCAN_API_KEY"
```

Comment:
I believe this explorer does not require a key? Can you confirm? Couldn't find where to get a key

Reply:
As I understand it, the API key can be obtained as the API replicates the etherscan API and there is mention of different tiers:

![image](https://github.com/ethereum/sourcify/assets/29679226/cb1ab905-6b21-4f6f-8986-da4f4f7ccf89)

Reply:
I couldn't find where to get a key. It seems to work fine without a key anyway

Reply:
Sure, you can merge it without the key)